packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -277,6 +278,8 @@ export function buildFrontendModuleName(plugin: PluginPackage | PluginModel): st
 export const HostedPluginClient = Symbol('HostedPluginClient');
 export interface HostedPluginClient {
     postMessage(message: string): Promise<void>;
+
+    hostedInstanceLog(logPart: LogPart): void;
```

Comment:
hello, simple question: as we're inside hosted plugin client interface, should we have hostedInstance in the name of the method ?

Reply:
Ok, I'll rename it.